runelite-client/src/main/java/net/runelite/client/plugins/mining/MiningPlugin.java

```diff
@@ -223,6 +253,43 @@ public void onGameTick(GameTick gameTick)
 		}
 	}
 
+	private void handleCamTorumStreamNotifications()
+	{
+		if (camTorumStreams.isEmpty() || !config.camTorumNotifyWaterSpawn())
+		{
+			return;
+		}
+		int ticksSinceNotif = client.getTickCount() - camTorumLastNotificationTick;
+		if (ticksSinceNotif < 52)
+		{ // streams last for about 45 or 50 game ticks
+			return; // skip, already notified for the current streams
+		}
+		camTorumLastNotificationTick = client.getTickCount();
+
+		boolean alreadyMiningStream = false;
+		boolean inCamTorumMine = false;
+		WorldPoint wp = client.getLocalPlayer().getWorldLocation();
+		for (Map.Entry<TileObject, Tile> entry : camTorumStreams.entrySet())
+		{
+			Tile tile = entry.getValue();
+			int distanceToStream = Math.abs(wp.getX() - tile.getWorldLocation().getX()) + Math.abs(wp.getY() - tile.getWorldLocation().getY());
```

Comment:
This formula is rather confusing, since it resembles an incorrectly-implemented version of  `WorldPoint#distanceTo(WorldPoint other)`. A comment explaining its purpose would help clarify this.